### PR TITLE
Update jackson-databind for sbt plugins

### PR DESCRIPTION
**Pull request by mdedetrich**
Resolves: https://github.com/apache/incubator-pekko/issues/376

### CODE REVIEW

**Comment by mdedetrich** [comment]
@pjfanning Seems like updating the jackson dependency didn't break anything

**Comment by pjfanning** [comment]
Still leaves us having to find out which plugin is causing the long dead jackson v1 (codehaus groupId) to be used

**Review by pjfanning — APPROVED** [review]
lgtm

**Comment by mdedetrich** [comment]
So I found out the issue, it appears we are using a no longer maintained https://github.com/spotify/docker-client which is bringing in the ancient jackson version. I have done a temporary workaround but we should replace/remove this test dependency (likely with something like testcontainers). Making an issue on it now.

File: project/Dependencies.scala [hunk 1]
```diff
@@ -184,6 +184,16 @@ object Dependencies {
       // docker utils
       val dockerClient = "com.spotify" % "docker-client" % "8.16.0" % Test
```
[the remark this answers is not in the document]
Comment (mdedetrich) [reply project/Dependencies.scala:185]: Which excludes?

[the remark this answers is not in the document]
Comment (He-Pin) [reply project/Dependencies.scala:185]: I mean `.exclude(...)`

**Review by He-Pin — APPROVED** [review]
lgtm

**Pull request merged**